tithemi/assignment1/Q2/Q2_linked_list.hpp

```diff
@@ -0,0 +1,58 @@
+template <typename T>
```

Comment:
What is the time complexity in the worst case of your solution?
Can it be faster?